src/modules/App/index.jsx

```diff
@@ -158,24 +158,24 @@ App.defaultProps = {
   dateLocale: null,
   allowProfileEdit: false,
   onProfileEditSuccess: null,
-  disableUserProfile: false,
   disableMarkAsDelivered: false,
   showSearchIcon: false,
   renderUserProfile: null,
   config: {},
-  isReactionEnabled: true,
-  isMentionEnabled: false,
-  isVoiceMessageEnabled: true,
   voiceRecord: {
     maxRecordingTime: VOICE_RECORDER_DEFAULT_MAX,
     minRecordingTime: VOICE_RECORDER_DEFAULT_MIN,
   },
-  replyType: 'NONE',
   isMessageGroupingEnabled: true,
   stringSet: null,
   colorSet: null,
   imageCompression: {},
   disableAutoSelect: false,
+  isMentionEnabled: false,
+  isReactionEnabled: true,
+  replyType: 'QUOTE_REPLY',
+  disableUserProfile: false,
+  isVoiceMessageEnabled: false,
```

Comment:
Just put every common configs \w dashboard ones together to see at a glance. 
Only replyType & isVoiceMessageEnabled value has been changed as I wrote in the description :)